charts/fluentbit/values.yaml

```diff
@@ -0,0 +1,158 @@
+namespace:
+  create : true
+  name: "calyptia-fluentbit"
+
+serviceAccount:
+  create: true
+
+image:
+  amd:
+    repository: http://709825985650.dkr.ecr.us-east-1.amazonaws.com/calyptia/calyptia-fb
+    tag: 22.10.2-amd64
```

Comment:
How are we going to automate the updates for this and the app version? Not for this PR but let's consider.